cmd/gctsDeployCommit.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"net/http/cookiejar"
+	"net/url"
+
+	gabs "github.com/Jeffail/gabs/v2"
+	"github.com/SAP/jenkins-library/pkg/command"
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/pkg/errors"
+)
+
+func gctsDeployCommit(config gctsDeployCommitOptions, telemetryData *telemetry.CustomData) {
+	// for command execution use Command
+	c := command.Command{}
+	// reroute command output to logging framework
+	c.Stdout(log.Entry().Writer())
+	c.Stderr(log.Entry().Writer())
+
+	// for http calls import  piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	// and use a  &piperhttp.Client{} in a custom system
+	// Example: step checkmarxExecuteScan.go
+	httpClient := &piperhttp.Client{}
+
+	// error situations should stop execution through log.Entry().Fatal() call which leads to an os.Exit(1) in the end
+	err := deployCommit(&config, telemetryData, &c, httpClient)
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+}
+
+func deployCommit(config *gctsDeployCommitOptions, telemetryData *telemetry.CustomData, command execRunner, httpClient piperhttp.Sender) error {
+
+	cookieJar, cookieErr := cookiejar.New(nil)
+	if cookieErr != nil {
+		return errors.Wrap(cookieErr, "deploy commit failed")
+	}
+	clientOptions := piperhttp.ClientOptions{
+		CookieJar: cookieJar,
+		Username:  config.Username,
+		Password:  config.Password,
+	}
+	httpClient.SetOptions(clientOptions)
+
+	URL := config.Host +
```

Comment:
This all caps looks odd to me, what about something like `requestUrl`?

Reply:
My Go format tool tells me it should be `requestURL` instead, but I will change to that if you prefer.